scripts/copy-monaco-assets.mjs

```diff
@@ -1,7 +1,5 @@
 import shell from 'shelljs'
 
 // copy workers to public
-shell.mkdir('-p', './public/monaco-editor-workers/workers');
-shell.cp('-fr', './node_modules/monaco-editor-workers/dist/index.js', './public/monaco-editor-workers/index.js');
-shell.cp('-fr', './node_modules/monaco-editor-workers/dist/workers/editorWorker-es.js', './public/monaco-editor-workers/workers/editorWorker-es.js');
-shell.cp('-fr', './node_modules/monaco-editor-workers/dist/workers/editorWorker-iife.js', './public/monaco-editor-workers/workers/editorWorker-iife.js');
+shell.mkdir('-p', './public/monaco-editor-wrapper/dist/workers');
+shell.cp('-fr', './node_modules/monaco-editor-wrapper/dist/workers/editorWorker-es.js', './public/monaco-editor-wrapper/dist/workers/editorWorker-es.js');
```

Comment:
No need for the IIFE format anymore?

Reply:
Firefox supports it now as well. Safari did for longer. I no longer see the need, finally. 🎉